funnel/models/project.py

```diff
@@ -178,63 +228,75 @@ def __init__(self, **kwargs):
     def __repr__(self):
         return '<Project %s/%s "%s">' % (self.profile.name if self.profile else "(none)", self.name, self.title)
 
-    state.add_conditional_state('HAS_PROPOSALS', state.POST_DRAFT,
+    state.add_conditional_state('HAS_PROPOSALS', state.PUBLISHED,
         lambda project: db.session.query(project.proposals.exists()).scalar(), label=('has_proposals', __("Has Proposals")))
-    state.add_conditional_state('HAS_SESSIONS', state.POST_DRAFT,
+    state.add_conditional_state('HAS_SESSIONS', state.PUBLISHED,
         lambda project: db.session.query(project.sessions.exists()).scalar(), label=('has_sessions', __("Has Sessions")))
```

Comment:
These two conditional states should be removed. Templates must use the other state managers.